data_model/src/lib.rs

```diff
@@ -839,8 +840,8 @@ pub mod model {
         Identifiable(IdentifiableBox),
         PublicKey(PublicKey),
         SignatureCheckCondition(SignatureCheckCondition),
-        TransactionValue(TransactionValue),
-        TransactionQueryResult(TransactionQueryResult),
+        TransactionValue(Box<TransactionValue>),
+        TransactionQueryResult(Box<TransactionQueryResult>),
```

Comment:
what happened here?

Reply:
Yeah, so now without `Box`'s compiler will fail to calculate `Value` size.

Explanation for `TransactionValue`
```
`<-`  means "contains"

Value <- TransactionValue <- VersionedRejectedTransaction <- RejectedTransaction <- TransactionRejectionReason <- ValidationFail <- InstructionExecutionFailure <- InstructionEvaluationError <- (new) TypeError <- Mismatch<Value>
```

Reply:
Assuming error is an exceptional case rather than the norm, maybe it would be more efficient to bring Box closer to the end of the chain?

Reply:
Makes sense

Reply:
Fixed